src/utilities/structured-output/json-structured-output.factory.ts

```diff
@@ -0,0 +1,169 @@
+import { OASResult, OperationExampleResult } from '../../results';
+import { StructuredOutput } from './structured-output.interface';
+
+class JSONStructuredOutputFactory {
+  static buildFromOASResult(input: OASResult): StructuredOutput {
+    const testResults = input.results;
+
+    // assemble authenticationType string
+    const authenticationType = input.securitySchemes
+      .map((x) => x.type)
+      .join('/');
```

Comment:
Since OAS supports multiple security schemes I'm thinking it might be better to maintain support for that in the JSON output and have docserver implement the temporary limitation.  Maybe by just using the first value if there are multiple.

I like the idea of making the security type an enum instead of a string (or an enum array).  Could we reuse the `OASSecurityType` definition from LOAST?  Maybe renaming some of the string values if needed?  [https://github.com/department-of-veterans-affairs/lighthouse-oas-tests/blob/master/src/utilities/oas-security/oas-security-scheme.ts](https://github.com/department-of-veterans-affairs/lighthouse-oas-tests/blob/master/src/utilities/oas-security/oas-security-scheme.ts)

Reply:
Got it. Using an enum array instead of the string sounds great to me! I think reusing the `OASSecurityType` enum sounds great. No need to change any of the string values, we can map those values to `apikey` or `oauth` on the Docserver side. 

I did see that `OASSecurityType` includes `MUTUAL_TLS`, but I don't see that in the OpenAPI Spec definition (attached screenshot). I think we should take that one out. We could do that as a separate ticket though - can maybe make sure we don't need it first

<img width="924" alt="Screen Shot 2022-06-23 at 5 35 58 PM" src="https://user-images.githubusercontent.com/4555395/175434572-815adb84-a268-46be-aa8d-6c57266774fc.png">

Reply:
Code and [documentation](https://community.max.gov/pages/viewpage.action?pageId=2286753455) updated so that `authenticationType` is now an array of enums.  Created a [new ticket](https://vajira.max.gov/browse/API-17188) for removing `MUTUAL_TLS`